l10n_it_ricevute_bancarie/tests/test_riba.py

```diff
@@ -536,3 +537,21 @@ def test_riba_presentation(self):
             self.env["account.move.line"].search(domain).mapped("amount_residual")
         )
         self.assertTrue(total_amount - total_issue_amount >= 0)
+
+    def test_riba_bank_multicompany(self):
+        with self.assertRaises(UserError) as ue:
+            self.env["riba.configuration"].create(
+                {
+                    "name": "Subject To Collection",
+                    "type": "incasso",
+                    "bank_id": self.company2_bank.id,
+                }
+            )
+        exc_message = ue.exception.args[0]
+        self.assertEqual(
+            "Incompatible companies on records:\n- "
+            f"'Subject To Collection' belongs to company '{self.env.company.name}' "
+            f"and 'Bank Account' (bank_id: '{self.company2_bank.acc_number}') "
+            "belongs to another company.",
+            exc_message,
+        )
```

Comment:
Non è un problema bloccante, ma credo che controllare l'uguaglianza esatta del testo dell'eccezione sia un po' troppo perché una qualsiasi modifica al testo nel core farà fallire il test.
Mi limiterei a controllare che vengano menzionate le aziende e i conti coinvolti.

Reply:
Ok, ho allineato in base a come hai indicato ed a come viene fatto in altri moduli,
Mi piaceva avere la certezza assoluta che un "assertIn" non può dare, ma sono consistenti anche le tue motivazioni.